subgraph-gnosis-chain.yaml

```diff
@@ -235,3 +257,49 @@ templates:
           handler: handleSwap
         - event: Sync(uint112,uint112)
           handler: handleSync
+  - kind: ethereum/contract
+    name: BalancerV1Pool
+    network: xdai
+    source:
+      abi: Pool
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      file: ./src/balancerV1/pool.ts
+      entities:
+        - User
+        - WeightedPool
+      abis:
+        - name: Pool
+          file: ./abis/balancerV1/BPool.json
+      eventHandlers:
+        - event: LOG_CALL(indexed bytes4,indexed address,bytes)
+          topic0: "0x34e1990700000000000000000000000000000000000000000000000000000000"
+          handler: handleSetSwapFee
+        # - event: LOG_CALL(indexed bytes4,indexed address,bytes)
+        #   topic0: "0x92eefe9b00000000000000000000000000000000000000000000000000000000"
+        #   handler: handleSetController
+        # - event: LOG_CALL(indexed bytes4,indexed address,bytes)
+        #   topic0: "0x49b5955200000000000000000000000000000000000000000000000000000000"
+        #   handler: handleSetPublicSwap
+        # - event: LOG_CALL(indexed bytes4,indexed address,bytes)
+        #   topic0: "0x4bb278f300000000000000000000000000000000000000000000000000000000"
+        #   handler: handleFinalize
+        # - event: LOG_CALL(indexed bytes4,indexed address,bytes)
+        #   topic0: "0x3fdddaa200000000000000000000000000000000000000000000000000000000"
+        #   handler: handleRebind
+        # - event: LOG_CALL(indexed bytes4,indexed address,bytes)
+        #   topic0: "0xe4e1e53800000000000000000000000000000000000000000000000000000000"
+        #   handler: handleRebind
+        # - event: LOG_CALL(indexed bytes4,indexed address,bytes)
+        #   topic0: "0xcf5e7bd300000000000000000000000000000000000000000000000000000000"
+        #   handler: handleUnbind
```

Comment:
```suggestion
```

cleanup

Reply:
You checked that none of those events are necessary?